src/clj_slackbot/comms/slack_web_hook.clj

```diff
@@ -50,11 +52,11 @@
     (go-loop [res (<!! cout)]
       (if-not res
         (println "The form output channel has been closed. Leaving listen loop.")
-        (let [result (:evaluator/result res)
-              channel (get-in res [:meta :channel])]
+        (let [channel (get-in res [:meta :channel])
+              post-url (get-in res [:meta :response-url])]
```

Comment:
Maybe destructuring would be a little better here 
`{{post-url :response-url channel :channel} :meta}`